outputs.tf

```diff
@@ -1,3 +1,7 @@
 output "tenant_data_access_email" {
   value = google_service_account.tenant_data_access.email
 }
+
+output "tenant_data_access_svc_account_id" {
+  value = google_service_account.tenant_data_access.id
+}
```

Comment:
Given this repository is public, shall we keep consistency between naming:
- tenant_data_access_email
- tenant_data_access_svc_account_id (_svc_ is what should go out or be added to top one)

Reply:
And maybe, what we should keep is just an ID, and remove Email exposer at all, because email could be obtained by data'ing svc by id.

Reply:
I think we should just output the svc account data object and then we can get id, name, email etc. 

if we just return id we can't get the email out and we'd have to start doing string manipulation which is a bit messy

Reply:
Makes sense